docs/customization/controller.rst

```diff
@@ -0,0 +1,167 @@
+Customizing Controllers
+=======================
+
+All **Sylius** resources are using the `` Sylius\Bundle\ResourceBundle\Controller\ResourceController`` as default, but some of them have been already extended in Bundles.
+If you want to override some controller action check which controller you should be extending.
+
+.. note::
+    There are two types of controllers we can define in Sylius.
+    **Resource Controllers** - are basing only on one Entity, so they return only the resources they have in their name. For instance ProductController should return only Products.
```

Comment:
I think we should use proper formatting for class names (like ``ProductController``), wdyt?